src/hugr/hierarchy.rs

```diff
@@ -1,14 +1,16 @@
-//! Region-level views of a HUGR.
```

Comment:
maybe the file/module should be called `views.rs`?

Reply:
I would like that, but there is already the `view.rs` file in the same folder that contains `HugrView` etc.

I could make a `views` folder with
- `views/hugr.rs`
- `views/hierarchy.rs`
in it?

Reply:
or just go with `hierarchical_views`